src/js/components/Chart/Chart.js

```diff
@@ -501,7 +510,11 @@ const Chart = React.forwardRef(
     const drawing = (
       <g
         stroke={stroke}
-        strokeWidth={type !== 'point' ? strokeWidth : undefined}
+        strokeWidth={
+          type !== 'point' && (type !== 'area' || !pattern)
```

Comment:
This change will behave strangely if the user tries out the type of 'bar' with a pattern. I agree this isn't something we should support since it isn't within the API, but since it still may impact the UI behavior, I would consider adding a warning for the usage of `pattern` without the type of 'area'.
Which leads me to another thought I had and wanted to share since `pattern` is only part of 'area', should we consider adding it to the area type as an object? It seems a little loud to have a whole prop for a singular use case of `type='area'`. so something like  `type={area: {pattern: '...'}}` or any other idea that will bind `pattern` directly to `area` and not let `pattern` float as an independent prop. I'm well aware that this idea may not be relevant in case we consider to allow pattern on other types than 'area', but thought I should share.

Reply:
I think this specific line will still behave correctly if the caller uses "bar" and "pattern", since `type != 'area'`.

I was thinking that we might want to support patterns for bars at some point down the road, so I was thinking to allow API growth in that direction. Also, we already have the `point` prop, which is currently only used when `type="point"`.